fix(testing-helpers): add `await elementUpdated(el)` supports stencil (#190)
open-wc/open-wc: packages/testing-helpers/README.md

```

## Timings
By default fixture awaits the elements "update complete" Promise.
- for [lit-element](https://github.com/polymer/lit-element) that is `el.updateComplete`;
- for [stencil](https://github.com/ionic-team/stencil/) that is `el.componentOnReady()`;

> If non of those specfic Promise hooks is found it will wait for one frame via `await nextTrame()`.

Reviewer: s/non/none/
s/nextTrame/nextFrame/

> If non of those specfic Promise hooks is found it will wait for one frame via `await nextTrame()`.

Reviewer 2: ```suggestion
If none of these specific Promise hooks are found, it will wait for one frame via `await nextFrame()`.
```

> Note that this is **NO** guarantee that the element is done rendering - it's "just" the next "frame".

Reviewer: ```
**Note**: this does not guarantee that the element is done rendering - it just waits for the next JavaScript tick.
```

> Basically fixture does that for you

Reviewer: ```
Essentially, `fixture` creates a synchronous fixture, then waits for the element to finish updating, checking `updateComplete` first, then falling back to `componentReady()`, and `nextFrame()` as a last resort.

This way, you can write your tests more succinctly, without having to explicitly `await` those hooks yourself.
```

```js
const el = fixtureSync(html`<my-el .foo=${'bar'}></my-el>`);
await elementUpdated(el);

> // which does the following:

Reviewer: Remove. put text in the text body, code in the snippet

// await el.updateComplete;
// OR await el.componentReady();
// OR await nextFrame();
expect(el.foo).to.equal('bar');

> // so you can instantly start testing without doing the awaits yourself

Reviewer: remove

```

### nextFrame

> Wait for the next frame via `requestAnimation`;

Reviewer: ```
Uses `requestAnimationFrame` to wait for the next frame.
